src/ops/math.jl

```diff
@@ -409,3 +409,15 @@ for reduction in [:sum, :prod, :min, :max, :mean]
         Tensor(Operation(desc), 1)
     end
 end
+
+@eval @op function unsorted_segment_sum(data::AbstractTensor,segment_ids,nbuckets; name=nothing)
+    if name === nothing
+        name = get_name("unsorted_segment_sum")
+    end
+    desc = NodeDescription("UnsortedSegmentSum", name)
+    segment_ids0=segment_ids-constant([1])
```

Comment:
I think mean to say `segment_ids0 = segment_ids .- 1`
0D scalar 1, rather than 1D vector  of length 1 containing 1

Spaces around operators (`+`) and `=` are generally nice.
Similar to the spaces after commas.